src/Appveyor.TestLogger/AppveyorApiTests.cs

```diff
@@ -0,0 +1,15 @@
+namespace Microsoft.VisualStudio.TestPlatform.Extensions.Appveyor.TestLogger
+{
+	internal class AppveyorApiTests
```

Comment:
Not needed since I'm building JSON manually instead of using a library. - Removed.